akka-docs/src/main/paradox/typed/persistence-style.md

```diff
@@ -0,0 +1,51 @@
+# Persistence - coding style
+
+## Event handlers in the state
+
+The section about @ref:[Changing Behavior](persistence.md#changing-behavior) described how commands and events
```

Comment:
A summary at the top with three bullets describing the different styles might be nice. Not mandatory.

Reply:
I think I leave that out for now. Not sure how this page will evolve, might include more style things, such as that lambdas in the javadsl builders should be one liners or method references.